confidant/services/jwkmanager.py

```diff
@@ -125,28 +130,28 @@ def get_jwt(self, environment: str, payload: dict,
             'expiry': expiry,
             'token': token
         }
-        stats.incr('get_jwt.create')
+        stats.incr('jwt.get_jwt.create')
         return token
 
     def get_active_key(self, environment: str) -> Tuple[str, Optional[jwk.JWK]]:
+        # The active signing key used to sign JWTs
         if environment in JWT_ACTIVE_SIGNING_KEYS and environment in self._keys:
-            return JWT_ACTIVE_SIGNING_KEYS[environment], self._get_key(
-                JWT_ACTIVE_SIGNING_KEYS[environment],
-                environment
-            )
+            kid = JWT_ACTIVE_SIGNING_KEYS[environment]
+            stats.incr(f'jwt.get_active_key.{environment}.{kid}')
```

Comment:
this is the new stats that we want.  it will allow us to see certificate key id used for signing